Update DigitalOcean how-to after v0.17 (#670)
meilisearch/documentation: resources/howtos/digitalocean_droplet.md

> ### 3. Enjoy your ready-to-use MeiliSearch droplet

Author: ```suggestion
### 3. Enjoy your ready-to-use MeiliSearch Droplet
```

> Your MeiliSearch droplet is ready to be used in production.

Reviewer: ```suggestion
Your MeiliSearch Droplet is ready to be used in production.
```

To check if everything is running smoothly, do an HTTP call to the health route:

```bash

> curl -v https://<your-meilisearch-url>/health

Reviewer: ```suggestion
$ curl -v https://<your-meilisearch-url>/health
```

```

> The server should answer with a `204 No content` ststus code as shown in teh example below:

Reviewer: ```suggestion
The server should answer with a `204 No content` status code as shown in the example below:
```
